frontend/src/components/_common/new-comment/NewComment.jsx

```diff
@@ -50,8 +81,20 @@ export default function NewComment({
 
   const handleSubmit = () => {
     if (!content) return;
-    onSubmit(content, isPrivate);
+    onSubmit(content.trim(), isPrivate);
     setContent('');
+    setUserTagList([]);
+  };
+
+  const handleOnSelectUserTag = (tag) => {
+    setContent((prev) => {
+      const lastTagQueryIndex = prev.lastIndexOf(`@${tagQuery}`);
+      const filteredContent = `${prev.substring(0, lastTagQueryIndex)}@${
+        tag.username
+      }`;
```

Comment:
github이나 instagram처럼 태그를 추가하면 뒤에 공백을 주는건 어떨까요?

```suggestion
      const filteredContent = `${prev.substring(0, lastTagQueryIndex)}@${
        tag.username
      } `;
```

그리고 태그 아이템 클릭 후에 다시 input에 focus를 주면 좋을것 같아요!
![Apr-22-2023 10-51-32](https://user-images.githubusercontent.com/48233023/233755641-1b4ec2e0-33c8-4d12-b8be-27664face513.gif)

Reply:
@July001 오우 좋아요! 반영해두었습니당!